src/core/plugins/json-schema-2020-12-samples/fn/class/OptionRegistry.js

```diff
@@ -4,7 +4,17 @@
 import Registry from "./Registry"
 
 class OptionRegistry extends Registry {
-  #defaults = {}
+  #defaults = {
+    randomEnabled: false,
+    random: Math.random,
+    minInt: 0,
+    maxInt: 2147483647,
+    minLen: 4,
+    maxLen: 10,
+    minDateTime: new Date("1970-01-01T00:00:00.000Z"),
+    maxDateTime: new Date("2038-01-19T00:00:00.000Z"),
```

Comment:
```suggestion
    maxDateTime: new Date("1970-01-01T00:00:01.000Z"),
```

Reply:
@char0n why would you not want any dates after 1970? Good to use json schema faker as inspiration but I would be wary of copying even the confusing parts?

Reply:
I just assumed that it solves some issue that we're currently not seeing. But no strong opinion here.